sql/wsrep_mysqld.h

```diff
@@ -131,7 +131,17 @@ enum enum_wsrep_mode {
   WSREP_MODE_REPLICATE_MYISAM= (1ULL << 3),
   WSREP_MODE_REPLICATE_ARIA= (1ULL << 4),
   WSREP_MODE_DISALLOW_LOCAL_GTID= (1ULL << 5),
-  WSREP_MODE_BF_MARIABACKUP= (1ULL << 6)
+  WSREP_MODE_BF_MARIABACKUP= (1ULL << 6),
+  WSREP_MODE_APPLIER_DISABLE_FK_WARNINGS= (1ULL << 7)
+};
+
+enum wsrep_warning_type {
+  WSREP_DISABLED = 0,
+  WSREP_REQUIRE_PRIMARY_KEY= 1,
+  WSREP_REQUIRE_INNODB= 2,
+  WSREP_EXPERIMENTAL= 3,
+  WSREP_APPLIER_FK_ERROR= 4,
+  WSREP_REQUIRE_MAX=5,
 };
```

Comment:
The `enum` name says `warning`, but one of the constants includes `ERROR`. It would be good to add a comment to each constant, explaining what kind of situation it is for.